From e229926c1bc8ec52e7ae89b975f9aa216ed1456e Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Thu, 21 Mar 2019 13:52:00 +0800
Subject: [PATCH 2/5] MLK-21123-2 arm64: fsl-imx8dx.dtsi: Correct DPU endpoint
 names

commit d17d5509736f359f9577afae0b7bb1e8025a5f28 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch corrects DPU endpoint names to suppress DTC warnings like below:
Warning (graph_endpoint): /dpu@56180000/port@0/lvds0-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@0/lvds1-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@0/mipi-dsi-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@1/lvds0-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@1/lvds1-endpoint: graph endpont node name should be 'endpoint'
Warning (graph_endpoint): /dpu@56180000/port@1/mipi-dsi-endpoint: graph endpont node name should be 'endpoint'

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Zhixiong Chi <zhixiong.chi@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index c9afad4e6ee0..0e33cf20dd73 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -1724,15 +1724,15 @@
 		dpu_disp0: port@0 {
 			reg = <0>;
 
-			dpu_disp0_lvds0_ch0: lvds0-endpoint {
+			dpu_disp0_lvds0_ch0: endpoint@0 {
 				remote-endpoint = <&ldb1_ch0>;
 			};
 
-			dpu_disp0_lvds0_ch1: lvds1-endpoint {
+			dpu_disp0_lvds0_ch1: endpoint@1 {
 				remote-endpoint = <&ldb1_ch1>;
 			};
 
-			dpu_disp0_mipi_dsi: mipi-dsi-endpoint {
+			dpu_disp0_mipi_dsi: endpoint@2 {
 				remote-endpoint = <&mipi_dsi1_in>;
 			};
 		};
@@ -1740,15 +1740,15 @@
 		dpu_disp1: port@1 {
 			reg = <1>;
 
-			dpu_disp1_lvds1_ch0: lvds0-endpoint {
+			dpu_disp1_lvds1_ch0: endpoint@0 {
 				remote-endpoint = <&ldb2_ch0>;
 			};
 
-			dpu_disp1_lvds1_ch1: lvds1-endpoint {
+			dpu_disp1_lvds1_ch1: endpoint@1 {
 				remote-endpoint = <&ldb2_ch1>;
 			};
 
-			dpu_disp1_mipi_dsi: mipi-dsi-endpoint {
+			dpu_disp1_mipi_dsi: endpoint@2 {
 				remote-endpoint = <&mipi_dsi2_in>;
 			};
 		};
-- 
2.17.1

